### PR TITLE
Github Actions: Lighthouse tests & npm audit

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**
- Run lighthouse tests in GH Actions

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: .github/workflows/simorgh-amp-a11y-validation.yml [hunk 1]
```diff
@@ -24,13 +24,20 @@ jobs:
         uses: actions/setup-node@v1
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-amp-a11y-validation.yml:24]: **nitpick:** is it worth renaming this file now given that we're grouping a number of tests which run when the server is running

**Review by karinathomasbbc — APPROVED** [review]

File: .github/workflows/simorgh-amp-a11y-validation.yml (hunk 1 above)
Comment (chris-hinds) [reply .github/workflows/simorgh-amp-a11y-validation.yml:24]: renaming it now

File: .github/workflows/simorgh-misc-checks.yml [hunk 2]
```diff
@@ -39,3 +39,6 @@ jobs:
 
       - name: Linting
         run: npm run test:lint
+
+      - name: NPM Audit
```
Comment (karinathomasbbc) [inline .github/workflows/simorgh-misc-checks.yml:43]: Can we also add Audit to the name at the top?

Comment (chris-hinds) [reply .github/workflows/simorgh-misc-checks.yml:43]: done

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**